src/main/java/seedu/address/model/RoomBook.java

```diff
@@ -15,10 +16,9 @@
     private static final Logger logger = LogsCenter.getLogger(JsonAddressBookStorage.class);
 
     private int numOfRooms;
-    private PriorityQueue<Room> rooms;
-    private Room[] roomsInArray;
-    private Path fileNumOfRooms;
-    private Path roomsOccupied;
+    private PriorityQueue<Room> rooms = new PriorityQueue<>();
+    private Room[] roomsInArray = new Room[0];
+
 
     /** Creates default RoomBook() object where all fields are null**/
     public RoomBook() {}
```

Comment:
May I ask what is the reason for naming the class `RoomBook` instead of `RoomList` or `RoomQueue`? I feel that `Book` does not explain what the class does.